documentation/asciidoc/topics/attributes/links.adoc

```diff
@@ -47,6 +47,8 @@ endif::community[]
 
 :link_os_olm: https://docs.openshift.com/container-platform/4.7/operators/understanding/olm-understanding-operatorhub.html[Understanding OperatorHub]
 
+:link_os_monitoring: https://access.redhat.com/documentation/en-us/openshift_container_platform/4.7/html-single/monitoring/index#enabling-monitoring-for-user-defined-projects[Enabling monitoring for user-defined projects]
```

Comment:
@oraNod seems to the same alias `:link_os_monitoring:` for different links locations. The second one should be removed

Reply:
nice catch